js/cryptoLib/index.js

```diff
@@ -1,29 +1,25 @@
 'use strict';
-const electron = require('electron');
-const app = electron.app;
 const path = require('path');
 const fs = require('fs');
 const lz4 = require('../compressionLib');
 const isDevEnv = require('../utils/misc.js').isDevEnv;
 const crypto = require('./crypto');
 const log = require('../log.js');
 const logLevels = require('../enums/logLevels.js');
+const searchConfig = require('../search/searchConfig.js');
 
-const userData = path.join(app.getPath('userData'));
-const DATA_FOLDER = isDevEnv ? './data' : path.join(userData, 'data');
-const INDEX_DATA_FOLDER = isDevEnv ? './data/search_index' : path.join(userData, 'data/search_index');
-const TEMPORARY_PATH = isDevEnv ? path.join(__dirname, '..', '..') : userData;
+const DUMP_PATH = isDevEnv ? path.join(__dirname, '..', '..') : searchConfig.FOLDERS_CONSTANTS.USER_DATA_PATH;
 
 class Crypto {
 
     constructor(userId, key) {
-        let INDEX_VERSION = 'v1';
-        this.indexDataFolder = INDEX_DATA_FOLDER + '_' + userId + '_' + INDEX_VERSION;
-        this.permanentIndexFolderName = 'search_index_' + userId + '_' + INDEX_VERSION;
-        this.dump = TEMPORARY_PATH;
+        this.indexDataFolder = searchConfig.FOLDERS_CONSTANTS.PREFIX_NAME_PATH +
```

Comment:
I would encourage using ES6 template literals here like how you've done below. They're cleaner.